wiki/Beatmap/Difficulty/id.md

```diff
@@ -5,21 +5,30 @@ tags:
+*Artikel ini tidak membahas seputar [Beatmap](/wiki/Beatmap) secara umum.*
 
-**Kesulitan** beatmap menggambarkan tingkat keterampilan yang diperlukan untuk menyelesaikannya. Nama tiap tingkat dan *star rating* dari sebuah beatmap biasanya memberikan informasi yang cukup untuk mengetahui seberapa sulit sebuah beatmap.
+Istilah **tingkat kesulitan beatmap** (*beatmap difficulty*) secara umum merujuk pada berkas-berkas [dengan ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)) yang tersimpan di dalam folder masing-masing beatmap. Berkas-berkas ini merupakan bentuk manifestasi dari level-level permainan yang ada dan berisikan antara lain informasi seputar [hit object](/wiki/Hit_object) pada [mode permainan](/wiki/Game_mode) yang terkait serta [timing](/wiki/Glossary#timing-section) dari lagu yang dipergunakan. Pada umumnya, seberapa sukar suatu tingkat kesulitan dapat dilihat dari nama dan [star rating](/wiki/Beatmapping/Star_rating) dari tingkat kesulitan tersebut.
 
-## Tingkat kesulitan
+Dalam ruang lingkup yang lebih spesifik, seberapa sukar suatu tingkat kesulitan turut dipengaruhi oleh hal-hal teknis seperti kompleksitas [pattern-pattern](/wiki/Beatmap/Pattern) yang ada dan [pengaturan tingkat kesulitan (*difficulty setting*)](/wiki/Beatmap_Editor/Song_Setup#difficulty) yang dipergunakan. Para pemain dapat bermain dengan memakai [modifier](/wiki/Game_modifier) untuk membuat suatu tingkat kesulitan menjadi lebih mudah atau lebih sukar untuk dimainkan.
+
+Di samping itu, suatu berkas tingkat kesulitan juga mengandung berbagai data yang terkait dengan sampel-sampel [hitsound](/wiki/Beatmapping/Hitsound) yang dapat didengar di dalam permainan.
+
+<!-- TODO: this description of visuals needs to be improved:
+  - a separate section;
+  - a very small paragraph on every key component, including those from Beatmap#overview;
+-->
+
+Dari sudut pandang visual, suatu berkas tingkat kesulitan dapat turut serta meliputi [berbagai komponen visual](/wiki/Beatmap) seperti [storyboard](/wiki/Storyboards) dan [skin](/wiki/Skinning) yang kemunculannya dapat diatur melalui menu [Visual Settings](/wiki/Visual_Settings). Di samping itu, terdapat pula berbagai informasi pendukung lainnya seperti waktu kemunculan [kiai time](/wiki/Kiai_time) dan [warna kombo](/wiki/Glossary/Combo_colour) yang ditampilkan.
+
+## Nama-nama tingkat kesulitan
 
 *Lihat juga: [Penamaan Tingkat Kesulitan](/wiki/Ranking_Criteria/Difficulty_Naming)*
 
-Berikut adalah nama tingkat kesulitan yang biasa dimiliki dalam beatmap. Definisi dan persyaratan khusus dari masing - masing tingkat dapat ditemukan dalam [kriteria beatmap Ranked](/wiki/Ranking_Criteria) untuk masing - masing mode.
+Berikut adalah nama-nama tingkat kesulitan yang umum dipergunakan dalam suatu beatmap. Definisi tiap-tiap tingkat kesulitan beserta panduan tata cara pemakaiannya masing-masing yang tepat guna dapat dilihat pada laman [Ranking Criteria](/wiki/Ranking_Criteria) masing-masing mode permainan.
```

Comment:
```suggestion
Berikut adalah nama-nama tingkat kesulitan yang umum dipergunakan dalam suatu beatmap. Definisi tiap-tiap tingkat kesulitan beserta panduan tata cara pemakaiannya masing-masing yang tepat guna dapat dilihat pada laman [Kriteria Ranking](/wiki/Ranking_Criteria) masing-masing mode permainan.
```

Reply:
yeah i also think this should be kept in English as well (notice how other locales for the Ranking Criteria i.e https://osu.ppy.sh/wiki/ko/Ranking_Criteria or https://osu.ppy.sh/wiki/zh/Ranking_Criteria retains "Ranking Criteria" in their article instead of translating it in their own language)

also similar to the above i can take a look at the current Ranking Criteria in the future to address this i suppose...